python/ray/_private/runtime_env/pip.py

```diff
@@ -189,9 +193,7 @@ def _create_or_get_virtualenv(cls, path: str, cwd: str, logger: logging.Logger):
                 virtualenv_path,
                 current_python_dir,
             )
-        exit_code, output = exec_cmd_stream_to_logger(
-            create_venv_cmd, logger, cwd=cwd, env={}
-        )
+        exit_code, output = exec_cmd_stream_to_logger(create_venv_cmd, logger, cwd=cwd)
```

Comment:
Remove the empty `env={}` which breaks windows (here and in other calls to `exec_cmd_stream_to_logger`)

Reply:
reverted this since it might be affecting linux